R/convertDiscrete.R

```diff
@@ -65,6 +65,6 @@ discreteValueToName = function(par, x) {
   if (par$type == "discrete") {
     ns[getIndex(par$values, x)]
   } else if (par$type == "discretevector")  {
-    sapply(x, function(x) ns[getIndex(x, values = par$values)])
+    vcapply(x, function(app  x) ns[getIndex(x, values = par$values)])
```

Comment:
You're right, I didn't consider the other points from which `discreteValueToName` gets called. I'd still say the printout of a `discreteVectorLearnerParam` with default `list()` shouldn't have `character(0)` in it.